### ISSUE TITLE
5-3 Logistic (regression) classifier tensor flow 로 구현

### ISSUE THREAD

**Issue by NahahaNahaha** [NahahaNahaha/tensorflow#11]

**Comment by NahahaNahaha** [comment]
x_data = [[1, 2], [2, 3], [3, 1], [4, 3], [5, 3], [6, 2]] 
y_data = [[0], [0], [0], [1], [1], [1]]
X = tf.placeholder(tf.float32, shape=[None, 2])
Y = tf.placeholder(tf.float32, shape=[None, 1])


W = tf.Variable(tf.random_normal([2, 1]), name='weight')
b = tf.Variable(tf.random_normal([1]), name='bias')
hypothesis = tf.sigmoid(tf.matmul(X, W) + b) //sigmod라는 함수는 tf.div(1., 1. + tf.exp(tf.matmul(X, W) + b))의 결과를 만들어준다. (Logistic regression 에서 함수)
cost = -tf.reduce_mean(Y * tf.log(hypothesis) + (1 - Y) * tf.log(1 - hypothesis))//이전 시간에 배웠던 cost식
train = tf.train.GradientDescentOptimizer(learning_rate=0.01).minimize(cost)//cost를 최소화 시키면서 W의 값을 변화시킨다. 지난번에 배웠던 미분에서 상수 곱한뒤 빼는 함수


predicted = tf.cast(hypothesis > 0.5, dtype=tf.float32)//여기서 hypothesis 가 0.5보다 크다면 predicted에 1이 저장, 아니라면 0이 저장된다. (cast 함수) 
accuracy = tf.reduce_mean(tf.cast(tf.equal(predicted, Y), dtype=tf.float32)) //만약 예측한 값이 실제값과 같다면 1 아니라면 0에서 그 평균을 구해 accuracy에 저장한다. 이 모델이 정확할수록 1에 가까울 것이고 아니라면 0에 가까울것이다. 

with tf.Session() as sess:
   sess.run(tf.global_variables_initializer())

   for step in range(10001):
       cost_val, _ = sess.run([cost, train], feed_dict={X: x_data, Y: y_data})
       if step % 200 == 0:
           print(step, cost_val)

   h, c, a = sess.run([hypothesis, predicted, accuracy],
                      feed_dict={X: x_data, Y: y_data})
   print("\nHypothesis: ", h, "\nCorrect (Y): ", c, "\nAccuracy: ", a) 
<img width="506" alt="2018-04-03 7 11 53" src="https://user-images.githubusercontent.com/26895961/38244059-570a8cd6-3774-11e8-98c9-2ae849853851.png">
그 결과다. 놀랍게도 정확도는 100, 데이터가 적어서 그런 것 같다.
